app/js/components/systemMetadata/systemMetadata.jade

```diff
@@ -1,11 +1,104 @@
-.row(ng-if='loading')
-  .col-sm-12
-    i.fa.spinner-xs.spinner
-    span Loading...
-.row.system-metadata(ng-if='!loading')
-  .col-sm-6(ng-repeat='facts in systemFacts')
-    table.meta-table.table-fixed
-      tbody
-        tr(ng-repeat='data in facts')
-          td(translate) {{data.label}}
-          td.metadata-value(translate) {{data.value}}
\ No newline at end of file
+.system-metadata(ng-if='!loading')
+  .flexgrid.equalheights
+    .row
+      // Hostname and UUID
+      .col-sm-7
+        .basic-information
+          .icon-inline
+            i.fa.fa-clone.fa-4x(ng-show="getSystemType() === 'Virtual'", tooltip="{{'Server type: Virtual' | translate}}")
+            i.fa.fa-server.fa-4x(ng-show="getSystemType() === 'Physical'", tooltip="{{'Server type: Physical' | translate}}")
+            i.fa.fa-question-circle.fa-4x(ng-show="getSystemType() === 'Unknown'", tooltip="{{'Server type: Unknown' | translate}}")
+
+            h2.hostname.ellipsis-overflow(translate, ng-if="system.hostname && !system.display_name") {{ system.hostname }}
+
+            // always show the uuid.... it must exist
+            .uuid.ellipsis-overflow
+              b UUID:&nbsp;
+              span(translate) {{ getUUID() }}
+            .alert-danger.text-center.pad5(ng-if='!system.isCheckingIn')
+              span(translate) This system is not checking in.
+
+      // Basic system meta-data table
+      .col-sm-5
+        table.meta-table
+          tbody
+            tr(ng-repeat='facts in initialMetadata')
+              td
+                b(translate) {{facts.label}}
+              td(translate) {{facts.value}}
+
+#expandableDataTable.expandable-data-table(ng-if='!loading; hasMetadata()', ng-class="{expanded: expandedTabs || expandIndefinite}")
+  // Toggle button
+  .toggle(ng-class="{expandedIndefinite: expandIndefinite}")
+    i.fa.fa-plus#iconSpin(ng-click="getData('network'); toggleTabsIndefinite(); rotateIcon()")
+
+  // Nav tabs
```

Comment:
When the user hovers over the ul, then the text for each tabs is revealed. Looks like: 
ul:hover li a .tabname { display: block; }

We can iterate over this if it needs  to be refined